tests/apis.js

```diff
@@ -36,96 +34,258 @@ test.cb.before(t => {
   });
 });
 
-test.cb.after.always(t => {
-  const items = types.map(type => {
-    return database('live').where('type', type).del().then(() => {
-      return database('schedule').where('type', type).del();
-    });
+/**
+ * Randomly selects a piece of content from source, and it's content-type model
+ *
+ * @param  {object} source object of content
+ * @returns {object}  - selected content and its model
+ */
+const testables = source => {
```

Comment:
I'd like to see this in the `fixtures` file instead of hanging out at the top of API tests. This way, we can reuse this all over the place (like in schedule)